pages/home.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<link rel="styleSheet" href="../style.css" />
+		<link rel="styleSheet" href="../layout/home.css" />
+
+		<title>Bookify</title>
+	</head>
+	<body>
+		<nav class="nav">
+			<label class="toggles">
+				<i class="fas fa-bars"></i>
+			</label>
+			<a class="nav_logo" href="./home.html"
+				><img
+					class="nav__logo-image"
+					alt="website logo"
+					src="../assets/Bookify.png"
+			/></a>
+			<div class="nav__input">
+				<input class="input" placeholder="Search For items" type="te" />
+				<i class="fa fa-search search__icon"></i>
+			</div>
+
+			<div class="nav__links">
+				<div class="nav__login">
+					<p>Hello User</p>
+					<button class="nav__login-btn">Login</button>
```

Comment:
The cursor remains the same on hovering, might be nice if it changed to pointer on button:hover

Reply:
Did the required  changes